server/data/restClient.ts

```diff
@@ -64,17 +65,32 @@ export default class RestClient {
     return this.config.timeout
   }
 
-  private logRequest(method: string, path: string, query = '') {
+  private logRequest(method: string, path: string, query = {}) {
     if (this.token) {
-      logger.info(`${method.toUpperCase()} request using user credentials: calling ${this.name}: ${path} ${query}`)
+      const queryString = this.queryString(query)
+      logger.info(`${method.toUpperCase()} request using user credentials: calling ${this.name}: ${path}${queryString}`)
```

Comment:
The queryString wasnt logged properly before (it was being logged as `[Object Object]`)